content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -239,6 +239,6 @@ You can use a selector which selects a subset or superset of the pods belonging
 controller.  However, when there are multiple PDBs in a namespace, you must be careful not
 to create PDBs whose selectors overlap.
 
-
-
-
+## {{% heading "whatsnext" %}}
+- PodDisruptionBudget is a top-level resource in the Kubernetes REST API.
```

Comment:
Check rendering. Do you need a line between the heading and the start of the list?